torch/onnx/_internal/diagnostics/infra/utils.py

```diff
@@ -41,13 +43,24 @@ def python_call_stack(frames_to_skip: int = 0, frames_to_log: int = 16) -> _infr
     return stack
 
 
+@functools.lru_cache()
```

Comment:
Do we need to set a size for the cache?

Reply:
I think it's by default 128? Seems good enough as it made this function disappear in profiler.

Reply:
Good catch I didn’t realize 128 was the default